From 04dcc9f6145e64c632babddbdcd1b6694ae9dbd1 Mon Sep 17 00:00:00 2001
From: glitch <glitch@users.noreply.github.com>
Date: Sat, 6 Mar 2021 06:59:55 -0500
Subject: [PATCH] GH-693 : Chapel patch for version 1.23.0 to get `chpldoc`
 target to build. The required changes update and sync the versions of sphinx
 etc.

---
 third-party/chpl-venv/chpldoc-requirements.txt                | 4 ++--
 .../chpl-venv/sphinxcontrib-chapeldomain/requirements.txt     | 4 ++--
 third-party/chpl-venv/sphinxcontrib-chapeldomain/setup.py     | 2 +-
 3 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/third-party/chpl-venv/chpldoc-requirements.txt b/third-party/chpl-venv/chpldoc-requirements.txt
index e73a957863..892d930190 100644
--- a/third-party/chpl-venv/chpldoc-requirements.txt
+++ b/third-party/chpl-venv/chpldoc-requirements.txt
@@ -2,12 +2,12 @@
 Jinja2==2.10.1
 MarkupSafe==1.1.1
 Pygments==2.4.2
-Sphinx==1.8.5
+Sphinx==3.2.1
 # Sphinx==2.0.1 # requires python 3
 # Sphinx==2.1.2 # requires python 3
 # Sphinx==2.2.0 # requires python 3
 docutils==0.15.2
 argparse==1.4.0
-sphinx-rtd-theme==0.4.3
+sphinx-rtd-theme==0.5.0
 ./sphinxcontrib-chapeldomain
 babel==2.7.0
diff --git a/third-party/chpl-venv/sphinxcontrib-chapeldomain/requirements.txt b/third-party/chpl-venv/sphinxcontrib-chapeldomain/requirements.txt
index 47d36821ed..e01b0f77a4 100644
--- a/third-party/chpl-venv/sphinxcontrib-chapeldomain/requirements.txt
+++ b/third-party/chpl-venv/sphinxcontrib-chapeldomain/requirements.txt
@@ -1,3 +1,3 @@
-docutils<=0.14
+docutils==0.15.2
 six
-Sphinx==1.8.5
+Sphinx==3.2.1
diff --git a/third-party/chpl-venv/sphinxcontrib-chapeldomain/setup.py b/third-party/chpl-venv/sphinxcontrib-chapeldomain/setup.py
index 78efb511a1..97eea973a0 100644
--- a/third-party/chpl-venv/sphinxcontrib-chapeldomain/setup.py
+++ b/third-party/chpl-venv/sphinxcontrib-chapeldomain/setup.py
@@ -44,7 +44,7 @@ setup(
     install_requires=[
         'docutils',
         'six',
-        'Sphinx>=1.6.0,<1.6.999',
+        'Sphinx>=3.2.1',
     ],
     namespace_packages=['sphinxcontrib']
 )
-- 
2.25.1

